bitnami/spring-cloud-dataflow/templates/server/deployment.yaml

```
@@ -168,7 +168,7 @@ spec:
           {{- end }}
           ports:
             - name: http
-              containerPort: 8080
+              containerPort: {{ .Values.server.port }}
```

Use `containerPort` instead:
```suggestion
              containerPort: {{ .Values.server.containerPort }}
```